spring-cloud-task-samples/batch-job/src/test/java/demo/BatchJobApplicationTests.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package demo;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.test.OutputCapture;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertTrue;
+
+/**
+ * Verifies that the Task Application outputs the correct task log entries.
+ *
+ * @author Michael Minella
+ */
+public class BatchJobApplicationTests {
+
+	@Rule
+	public OutputCapture outputCapture = new OutputCapture();
+
+	@Test
+	public void testTimeStampApp() throws Exception {
+		final String JOB_RUN_MESSAGE = " was run";
+		final String CREATE_TASK_MESSAGE = "Creating: TaskExecution{executionId=";
+		final String UPDATE_TASK_MESSAGE = "Updating: TaskExecution{executionId=";
+		final String JOB_ASSOCIATION_MESSAGE = "The job execution id ";
+
+		assertEquals(0, SpringApplication.exit(SpringApplication
+				.run(BatchJobApplication.class)));
+
+		String output = this.outputCapture.toString();
+		assertTrue("Unable to find the timestamp: " + output,
+				output.contains(JOB_RUN_MESSAGE));
+		assertTrue("Test results do not show create task message: " + output,
+				output.contains(CREATE_TASK_MESSAGE));
+		assertTrue("Test results do not show success message: " + output,
```

Comment:
Have to update tests to support that Update log message is only debug now.  Will do this on merge.